### ISSUE TITLE
Board closed - head to trello

### ISSUE THREAD

**Issue by mdshearer** [enspiral/improvements#640]
Improvements board now lives in [Trello](https://trello.com/b/btHeb35m/enspiral-improvements-board)

**Comment by patcon** [comment]
And just to ask for anyone curious, it's private now, right?

Also, might want to remove/update the broken "key link" in the footer to the waffle board :)

**Comment by mdshearer** [comment]
Accidentally private... but no decision on whether it *should* be or not!

**Comment by mdshearer** [comment]
@coopchange @joshuavial Or anyone else... Thoughts on whether trello improvements board should be made public like Github was...?

**Comment by phillcoxon** [comment]
I vote public and we should update the readme for this repo to point to the new trello.

**Comment by coopchange** [comment]
++ @phillcoxon 's comment

BTW Phill, do you know how to re-orient improvements.enspiral.com to [the trello's url](https://trello.com/b/btHeb35m/enspiral-improvements-board)? I felt like you or JV are the go to people on subdomains/ redirects etc.

**Comment by patcon** [comment]
Re: subdomains. I'm inspired by this, and looking to implement it for governance of some similarly shared domains. (GitHub Actions can make updates even more clear, if you trust it :) )

https://github.com/g0v/domain

Happy to work on it myself if you let me know the DNS provider you use -- we haven't committed to one yet. This is prob the kick in the pants I need to implement it for our orgs too! Re: @hyphacoop @civictechto

(Can re-ticket in a new issue)

tl;dr - manage domain records from repo config via pull request. Assign per-record stewards who "own" the subdomain, and have that info clearly in the public config.

**Comment by coopchange** [comment]
Hi @patcon glad to hear you're inspired and eager to exchange, but to be honest I'm feeling I'm missing some understanding of what you're offering. Maybe someone a bit more git/tech than me will want to pick up on this…

@mdshearer – do you need anything from me to proceed? Or maybe you have idea who could accomplish the redirect?

I can update the Readme to point to the Trello

**Comment by mdshearer** [comment]
@joshuavial Do you have the keys to the DNS for redirects?

**Comment by patcon** [comment]
@coopchange Thanks for the charitable response in the face of my poor comms :)

I'll implement the process/tool for my own organization and then have something to show. If you see any value demonstrated, I'm happy to pick up convo at that time 😉
